### PR TITLE
cmd/bigslice: set DefaultRegion and cleanup error message

### PR DESCRIPTION
**Pull request by cosnicolaou**
This PR makes use of the change made [in](https://github.com/grailbio/bigmachine/pull/29) to set a default AWS region if one is present in the bigmachine config. It also improves an error message.

Also, dependencies are updated for grailbio/base to v.0.0.6 and not .0.0.5

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,11 @@ go 1.12
...
 	github.com/google/gofuzz v1.0.0
-	github.com/grailbio/base v0.0.5
+	github.com/grailbio/base v0.0.6
 	github.com/grailbio/bigmachine v0.5.5
 	github.com/grailbio/testutil v0.0.3
 	github.com/spaolacci/murmur3 v1.1.0
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 )
+
+replace github.com/grailbio/bigmachine => /data/forks/bigmachine
```
Comment (jcharum) [inline go.mod]:
```suggestion
```

File: go.mod [hunk 2]
```diff
@@ -5,9 +5,11 @@ go 1.12
...
 	github.com/aws/aws-sdk-go v1.25.13
 	github.com/google/gofuzz v1.0.0
-	github.com/grailbio/base v0.0.5
+	github.com/grailbio/base v0.0.6
 	github.com/grailbio/bigmachine v0.5.5
 	github.com/grailbio/testutil v0.0.3
 	github.com/spaolacci/murmur3 v1.1.0
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 )
+
```
Comment (jcharum) [inline go.mod]:
```suggestion
```

File: go.mod (hunk 1 above)
Comment (jcharum) [reply go.mod]: Accidentally included?

File: cmd/bigslice/ec2.go [hunk 3]
```diff
@@ -74,6 +75,10 @@ func setupEc2Cmd(args []string) {
 		must.True(os.IsNotExist(err), err)
 	}
 
+	if region, ok := profile.Get("aws/env.region"); ok && len(region) > 0 {
+		must.Nil(profile.Set("bigmachine/ec2system.default-region", strings.Trim(region, `"`)))
```
Comment (jcharum) [inline cmd/bigslice/ec2.go]: Why do we need to `Trim` here?

(Sorry I missed this the first time around).

Comment (cosnicolaou) [reply cmd/bigslice/ec2.go]: Json double-escaping... the region can end up being "\"us-west-2\"".

Comment (jcharum) [reply cmd/bigslice/ec2.go]: Won't that only happen if we muck up escaping somewhere else?  It seems that we [do something similar with`ec2system.security-group`](https://github.com/grailbio/bigslice/blob/0d07a334180d1918b8bbf594e14628ceacdd1745/cmd/bigslice/ec2.go#L82), though I'm not sure why there either.

Comment (cosnicolaou) [reply cmd/bigslice/ec2.go]: I don't really know all of the code paths that lead to a value being setup here, nor how quotes are expected to be handled by grailbio/config but it's certainly the case the escaped quotes end up here. So in the interests of expediency, and knowing that the region will not have internal quotes, it seemed reasonable to just remove them here.

**Pull request merged**